frontend/src/components/object/ObjectUpload.vue

```diff
@@ -44,7 +45,9 @@ const onUpload = async (event: any) => {
     await Promise.allSettled(
       event.files.map(async (file: File) => {
         try {
-          await objectStore.createObject(file, bucketId);
+          //infinite timeout for big files upload to avoid timeout error
+          const axiousOptions: AxiosRequestConfig = { timeout: 0 };
+          await objectStore.createObject(file, bucketId, axiousOptions);
```

Comment:
Typo with const name. Should be `axiosOptions`.
Also, as this is really only being used in one place, I believe you can probably in-line this to avoid needing to do a type import as well:
```ts
await objectStore.createObject(file, bucketId, { timeout: 0 });
```